### ISSUE TITLE
func BooleanFilterToMods has empty body

### ISSUE THREAD

**Issue by troian** [web-ridge/gqlgen-sqlboiler#22]
```go
func BooleanFilterToMods(m *graphql_models.BooleanFilter, column string) []qm.QueryMod {

	return nil
}
```

**Comment by RichardLindhout** [comment]
That's right can you make a PR for that? I did not have boolean usecase yet so not fully sure how to make working queries for it

**Comment by RichardLindhout** [comment]
This should probably work

```
func BooleanFilterToMods(m *graphql_models.BooleanFilter, column string) []qm.QueryMod {
	if m == nil {
		return nil
	}
	var queryMods []qm.QueryMod
	if m.IsFalse != nil {
		queryMods = append(queryMods, qmhelper.Where(column, qmhelper.EQ, false))
	}
	if m.IsTrue != nil {
		queryMods = append(queryMods, qmhelper.Where(column, qmhelper.EQ, true))
	}
	if m.IsNull != nil {
		queryMods = append(queryMods, qmhelper.Where(column, qmhelper.EQ, nil))
	}
	return queryMods
}
```

**Comment by troian** [comment]
The Sample you've provided exactly same what we've ended up with)

**Comment by RichardLindhout** [comment]
Are you in favor of changing input in scheme to: 
```
input BooleanFilter {
  equalTo: Boolean
  notEqualTo: Boolean
}
```

**Comment by troian** [comment]
Yah, sounds good

**Comment by RichardLindhout** [comment]
Fixed in v2.0.12
